manifests/d/Docker/DockerDesktop/4.0.0/Docker.DockerDesktop.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: Docker.DockerDesktop
+PackageVersion: 4.0.0
+PackageLocale: en-US
+Publisher: Docker Inc.
+PublisherUrl: https://www.docker.com
+PrivacyUrl: https://www.docker.com/legal/privacy
+Author: Docker Inc.
+PackageName: Docker Desktop
+PackageUrl: https://www.docker.com
+License: Apache License 2.0
+LicenseUrl: https://www.docker.com/legal/docker-software-end-user-license-agreement
+Copyright: Copyright © 2015-2021 Docker Inc. All rights reserved.
+ShortDescription: Docker Desktop is an application for macOS and Windows machines for the building and sharing of containerized applications. Access Docker Desktop and follow the guided onboarding to build your first containerized application in minutes.
+Moniker: docker
+Tags:
+- Container
+- Containerization
+- Virtualization
```

Let's change this so it follows the best practices going forward [1.0.0/defaultLocale.md#tags](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md#tags)

```suggestion
- container
- containerization
- virtualization
```